test/tools/README.testnames

```diff
@@ -20,15 +20,15 @@ n is the configuration type:
 6 -- unused
 7 -- unused
 8 -- unused
-9 -- unused
+9 -- mesh_maker
```

Comment:
I add some test/tools testing here, which is in the process of being deprecated. So maybe doesn't need to come in. But, it also doesn't seem like it hurts to have it either...